docker-compose.yml

```
@@ -0,0 +1,30 @@
+version: '${TAG_COMPOSE}'
+services:
+  db:
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_HOST_AUTH_METHOD: trust
+    image: 'postgres:${TAG_POSTGRES}'
+    ports:
+      - '5432:5433'
+    restart: always
+    volumes:
+      - 'pgdata:/var/lib/postgresql/data'
```

@keliumJU let's reorder this like this:
```suggestion
  mintacoin_db:
    image: postgres:14-alpine
    container_name: mintacoin_db
    restart: always
    volumes:
      - pgdata:/var/lib/postgresql/data
    environment:
      - POSTGRES_USER=postgres
      - POSTGRES_PASSWORD=postgres
      - POSTGRES_DB=mintacoin_dev
    networks:
      - mintacoin
```

---

ok, change made